contributors/devel/sig-node/triage.md

```diff
@@ -0,0 +1,128 @@
+# SIG Node Triage Process
+
+Currently, SIG Node is limiting its triage review to pull requests. In the
+near-future, we hope to expand to issues as well.
+
+This is intended to guide developers through the SIG Node triage process. You
+will learn about the roles of authors, reviewers, and approvers, as well as
+what labels are required for each stage of the PR process.
+
+We track most node PRs on the [SIG Node Triage project board]. All test and
+CI-related PRs are tracked on the [CI subproject board].
+
+[SIG Node Triage project board]: https://github.com/orgs/kubernetes/projects/49
+[CI subproject board]: https://github.com/orgs/kubernetes/projects/43
+
+## Triage
+
+All new PRs added to the board should begin in the **Triage** column.
+
+When a pull request is made against kubernetes/kubernetes, it will typically
+have the following [labels]:
+
+- needs-triage
+- needs-priority
+- needs-ok-to-test (not needed for project members)
+
+In order to be moved out of this column, all of the above labels must be set.
+
+[labels]: https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md
+
+### needs-ok-to-test
+
+Use your best judgment in determining whether a PR is OK to test. You don't
+have to do a full review: just make sure that the code does not appear to be
+actively malicious, and the PR appears to be doing something useful. Use the
+command `/ok-to-test`.
+
+If the PR is trivial and doesn't provide much value, feel free to close it and
+link the author to the [trivial edits policy].
+
+[trivial edits policy]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits
+
+### needs-priority
+
+You can take a quick look at what the PR is addressing and then apply a
+priority label.
+
+- **priority/critical-urgent:** Urgent bug fix, required ASAP.
```

Comment:
`critical-urgen` also tells the release team that they can't make a release.

Reply:
if it's critical-urgent feels like maybe we should do something more than just tag it? maybe also post in sig-node channel to flag?